packages/create-widget/src/run.ts

```diff
@@ -30,8 +30,8 @@ interface RunArgs {
   sdkVersion: SdkVersion;
   foundryUrl: string;
   widgetSet: string;
-  osdkPackage: string;
-  osdkRegistryUrl: string;
+  osdkPackage?: string;
+  osdkRegistryUrl?: string;
```

Comment:
Is there anywhere that runs the CLI without these args to test that they actually aren't required? Maybe in `cli.test.ts` or example-generator